bibliography.bib

```diff
@@ -7,6 +7,68 @@ @inproceedings{gullasch2011cache
   organization={IEEE}
 }
 
+@inproceedings{osvik2006cache,
+  title={Cache attacks and countermeasures: the case of {AES}},
+  author={Osvik, D. A. and Shamir, A. and Tromer, E.},
+  booktitle={RSA Conference Cryptographers Track (CT-RSA) 2006},
+  year={2006}
+}
+
+@inproceedings{tromer2009cache,
+  title={Efficient cache attacks on {AES}, and countermeasures},
+  author={Tromer, E. and Osvik, D. A. and Shamir, A.},
+  booktitle={Journal of Cryptology},
+  year={2009},
+}
+
+@incollection{ristenpart2009cloud,
+  title={Hey, you, get off of my cloud: exploring information leakage in third-party compute clouds},
+  author={Ristenpart, Thomas and Tromer, Eran and Shacham, Hovav and Savage, Stefan},
+  booktitle={CCS '09 Proceedings of the 16th ACM conference on Computer and communications security},
+  pages={199--212},
+  year={2009},
+  publisher={ACM},
+  annote={
+    \setlength{\parskip}{1.5ex}
+    Describes a method for achieving co-residence with target VMs on the cloud and exploiting information leakage between these VMs using 
+    Amazon's EC2 service as an example.
```

Comment:
Better, but also indent sentence continuations an extra notch. I know I'm being picky, but future Nate will thank you for the readability improvement.